turbinia/config/turbinia_config.py

```diff
@@ -45,28 +82,27 @@
 # NFS or a SAN for Evidence objects.
 SHARED_FILESYSTEM = False
 
-# This will set debugging flags for processes executed by Tasks (for
-# Tasks/binaries that support it).  This could cause performance issues with
-# some tasks, so it is recommended to only set this to True when debugging
-# problems.
-DEBUG_TASKS = False
+###############
+# GCP AND GCE #
+###############
 
-# GCE configuration
 PROJECT = None
 ZONE = None
 INSTANCE = None
 DEVICE_NAME = None
 SCRATCH_PATH = None
-BUCKET_NAME = None
+BUCKET_NAME = 'turbinia'
 PSQ_TOPIC = 'turbinia-psq'
 
-# Topic Turbinia will listen on for new Artifact events.  This is also used as
+# Topic Turbinia will listen on for new Artifact events. This is also used as
 # the Turbinia instance/namespace as it is a unique string per Turbinia
 # instance and Cloud Project.
-PUBSUB_TOPIC = None
+PUBSUB_TOPIC = 'turbinia-pubsub'
 
 # GCS Path to copy worker results and Evidence output to
-GCS_OUTPUT_PATH = False
+# This is required if OUTPUT_DIR is pointing to a GCSFuse mount.
```

Comment:
Actually, this is independent of whether the OUTPUT_DIR is pointing to GCSFuse.  I believe the only thing this is used for is task output.   I don't think we need this line.

Reply:
turbinia/output_manager.py is using GCS_OUTPUT_PATH. Do we still need that line?

Reply:
Sorry, I meant this comment line.  We definitely still want the config variable.

Reply:
Done